### PR TITLE
Dynatrace Connector Certification Submission

### PR DESCRIPTION
**Pull request by mandy-swinton**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Dynatrace/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          "x-ms-apimTemplateParameter.name": "from",
+          "x-ms-apimTemplateParameter.value": "now-1w",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-operationName": [
+            "GetProblems"
+          ]
+        }
+      }
+    ],
+    "publisher": "Swinton, Mandy",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynatrace/apiProperties.json:54]: @mandy-swinton please make sure the publisher is specified according to [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). It should be the name of company or organization.

File: certified-connectors/Dynatrace/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,982 @@
...
+                },
+                "managementZones": {
+                  "type": "array",
+                  "items": {},
+                  "description": "managementZones"
+                },
+                "entityTags": {
+                  "type": "array",
+                  "items": {},
+                  "description": "entityTags"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynatrace/apiDefinition.swagger.json:552]: @mandy-swinton please fix the description, it should be like `Entity tags`, but not in lowerCamelCase. Look through the others too please.

File: certified-connectors/Dynatrace/Readme.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Dynatrace Connector
+
+Dynatrace provides best-in-class observability through an open, AI-powered platform. This connector exposes the Problems API endpoints. For more details on the Problems API, check out the documentation here: https://www.dynatrace.com/support/help/dynatrace-api/environment-api/problems-v2
+
```
[the remark this answers is not in the document]
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/Readme.md:4]: Added publisher information as well as other required fields

File: certified-connectors/Dynatrace/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,982 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Dynatrace",
+    "description": "Dynatrace provides best-in-class observability through an open, AI-powered platform. This connector contains the endpoints for the Dynatrace Problems API.",
```
[the remark this answers is not in the document]
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiDefinition.swagger.json]: Removed "API" from the description

File: certified-connectors/Dynatrace/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,982 @@
...
+                    }
+                  },
+                  "description": "impactAnalysis"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GetProblemById",
+        "summary": "Get Problems by ID",
```
[the remark this answers is not in the document]
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiDefinition.swagger.json]: You are correct. Updated.

File: certified-connectors/Dynatrace/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,982 @@
...
+        "description": "Gets the properties of the specified problem.",
+        "parameters": [
+          {
+            "name": "problem_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Problem Id",
```
[the remark this answers is not in the document]
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiDefinition.swagger.json]: I have updated all instances of ID and any other abbreviations I could find

File: certified-connectors/Dynatrace/apiProperties.json (hunk 1 above)
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiProperties.json]: I have updated the publisher

File: certified-connectors/Dynatrace/apiDefinition.swagger.json (hunk 2 above)
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiDefinition.swagger.json]: I have updated all the descriptions to not be in camel case

File: certified-connectors/Dynatrace/Readme.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (v-EgorKozhadei) [reply certified-connectors/Dynatrace/Readme.md:4]: Please look through once more, the `Deployment Instructions` heading is missing.

[the remark this answers is not in the document]
Comment (v-EgorKozhadei) [reply certified-connectors/Dynatrace/Readme.md:4]: Please look through once more, the `Deployment Instructions` heading is missing.

File: certified-connectors/Dynatrace/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,982 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Dynatrace",
+    "description": "Dynatrace provides best-in-class observability through an open, AI-powered platform. This connector contains the endpoints for the Dynatrace Problems interface.",
+    "version": "1.0",
+    "contact": {
+      "name": "Connector Support",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynatrace/apiDefinition.swagger.json:8]: @mandy-swinton please change this one to `Dynatrace Support`, thank you!

File: certified-connectors/Dynatrace/Readme.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/Readme.md:4]: Added

File: certified-connectors/Dynatrace/apiDefinition.swagger.json (hunk 7 above)
Comment (mandy-swinton) [reply certified-connectors/Dynatrace/apiDefinition.swagger.json]: Updated

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @mandy-swinton, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**